BAZProject/InformationMovies/View/InformationMovieViewCollectionProtocolsExtension.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  InformationMovieViewCollectionProtocolsExtension.swift
+//  BAZProject
+//
+//  Created by 1014600 on 11/02/23.
+//
+
+import UIKit
+
+// MARK: -  UICollectionViewDataSource
+extension InformationMoviesView: UICollectionViewDataSource {
+    func numberOfSections(in collectionView: UICollectionView) -> Int {
+        return Constants.numberOfSections
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        if presenter?.getSimilarMoviesCount() ?? 0 > 6 {
+            return 6
+        }
+        return presenter?.getSimilarMoviesCount() ?? 0
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        guard let cell = collectionView.dequeueReusableCell(withReuseIdentifier: MovieCollectionSimilarCell.cellIdentifier, for: indexPath) as? MovieCollectionSimilarCell else {
+            return UICollectionViewCell()
+        }
+        
+        cell.setupCell(movie: self.presenter?.getSimilarMovie(indexPathRow: indexPath.row) ?? Movie())
```

Comment:
ya que estas agregando tu cell a un Guard let podrias meter igual la validación de tu row en el mismo 
`guard let cell = collectionView.dequeueReusableCell(withReuseIdentifier: MovieCollectionSimilarCell.cellIdentifier, for: indexPath) as? MovieCollectionSimilarCell,
let row = self.presenter?.getSimilarMovie(indexPathRow: indexPath.row) else { return .....`